spec/AutomatedLinkAcceptanceProxy.yaml

```diff
@@ -1436,17 +1436,15 @@ paths:
                     type: string
                     example: 'LTP-MWPS-TTP-1-1'
                   description: >
-                    'The UUID of the LogicalTerminationPoint that is addressed by the request
-                    from UuidUnderTest determined in RequestForProvidingAcceptanceDataCausesDeterminingAirInterfaceUuidUnderTest'
+                    'The UUID of a LogicalTerminationPoint with LayerProtocolName == AIR_LAYER'
```

Comment:
Shall we make the AIR_LAYER to 'air-interface-2-0:LAYER_PROTOCOL_NAME_TYPE_AIR_LAYER' (since we mentioned a "=="). Many thanks.
(same comment applicable to the next change aswell)

Reply:
Changed AIR_LAYER to the complete LayerProtocolName